src/command/schema/fetch.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
+use houston as config;
+use rover_client::blocking::Client;
+use rover_client::query::schema::get;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct Fetch {
+    /// ID of the graph to fetch from Apollo Studio
+    #[structopt(name = "SCHEMA_ID")]
+    schema_id: String,
+    /// The variant of the request graph from Apollo Studio
+    #[structopt(long, default_value = "current")]
+    variant: String,
+    #[structopt(long, default_value = "default")]
+    profile: String,
```

Comment:
```suggestion
    #[structopt(long = "profile", default_value = "default")]
    profile_name: String,
```

this is consistent with the profile argument we use in `src/command/config/api_key`.

i wonder if there is a way to store these commands with `StructOpt` so it's easier to keep them consistent